pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -41,6 +50,17 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 
 	url := fmt.Sprintf("%s/rest/api/orgs/%s/analytics", config.GetString(configuration.API_URL), config.Get(configuration.ORGANIZATION))
 
+	commandLineInput := config.GetString(reportAnalyticsInputDataFlagName)
+	if commandLineInput != "" {
+		logger.Printf(fmt.Sprintf("%s: adding command line input", reportAnalyticsWorkflowName))
```

Comment:
nitpick: you don't have to prefix log messages, the logger already uses a prefix.